app/utils/wrap-downloaded-document.js

```diff
@@ -0,0 +1,13 @@
+export function wrapDownloadedDocument(html) {
+  const document = `
+      <html>
```

Comment:
might as well be proper here and add the doctype? https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#anatomy_of_an_html_document